product_docs/docs/pgd/5/reference/repsets-membership.mdx

```diff
@@ -8,10 +8,9 @@ indexdepth: 2
 
 This function adds a table to a replication set.
 
-This adds a table to a replication set and starts replicating changes
-from this moment (or rather transaction commit). Any existing data the table
+This function adds a table to a replication set and starts replicating changes
+at that time (on transaction commit). Any existing data the table
```

Comment:
Is this what was meant? It took reading this a few times to glean what was meant.